post-scan-actions/aws-python-publish-malware-to-sns/template.yml

```diff
@@ -0,0 +1,90 @@
+AWSTemplateFormatVersion: 2010-09-09
+Transform: AWS::Serverless-2016-10-31
+
+Metadata:
+  AWS::ServerlessRepo::Application:
+    Name: cloudone-filestorage-plugin-publish-malware-to-sns
+    Description: >-
+      According to the scan result from the scanner, publish the malware information
+      to the target SNS topic.
+    Author: Trend Micro Cloud One File Storage Security
+
+Parameters:
+  TargetSnsTopicName:
+    Type: String
+    Description: The name for the SNS topic to which malware information publish.
```

Comment:
The customer may already have their own SNS topic. Could you figure out a way for using the existing SNS topic?

Reply:
I will add 1 more parameter to let user fill in their customer SNS Topic ARN, and create a new one SNS topic if the value is empty string.